### PR TITLE
Make Schema Changes to --worker-info Command, Surface UUID

### PR DESCRIPTION
**Pull request by beeankha**
Connecting https://github.com/ansible/ansible-runner/issues/788

Schema changes are being made in order to more accurately reflect what information is getting surfaced with the `--worker-info` command.

### CODE REVIEW

**Comment by AlanCoding** [comment]
:+1: looking good so far, still testing.

**Comment by AlanCoding** [comment]
@beeankha this is a really small amount of memory!

```
$ ansible-runner worker --worker-info
{cpu_count: 8, errors: [], mem_in_bytes: 32619828, runner_version: 2.0.0, uuid: ade28af7cbce4360a7bf2c4d45de43a1}
```

**Review by AlanCoding — APPROVED** [review]

File: ansible_runner/__main__.py [hunk 1]
```diff
@@ -766,17 +766,24 @@ def main(sys_args=None):
...
+                errors.append(mem)
                 mem = None
-            info = {'Errors': error,
-                    'Memory Capacity': mem,
-                    'CPU Capacity': cpu,
-                    'Version': VERSION,
+            if "Could not find" in uuid:
+                errors.append(uuid)
+                uuid = None
+            else:
```
Comment (jladdjr) [inline ansible_runner/__main__.py]: this looks like just a no-op

**Review by jladdjr — APPROVED** [review]

**Comment by AlanCoding** [comment]
passing for me locally

```
py.test test/unit/test_runner.py::test_env_vars[abc123]
```

with this branch. Maybe test flake, unsure.

**Pull request merged**